### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by abejgonzalez**

### CODE REVIEW

File: docs/Simulation/FPGA-Based-Simulators.rst [hunk 1]
```diff
@@ -3,11 +3,15 @@ FPGA-Based Simulators
...
+`FireSim <https://fires.im/>`__ is an open-source cycle-accurate FPGA-accelerated full-system hardware simulation platform that runs on cloud FPGAs (Amazon EC2 F1).
+FireSim allows RTL-level simulation at orders-of-magnitude faster speeds than software RTL simulators.
+FireSim also provides additional device models to allow full-system simulation, including memory models and network models.
 
-Follow the initial EC2 setup instructions as detailed in the `FireSim documentation  <http://docs.fires.im/en/latest/Initial-Setup/index.html>`__ .. Then clone your full ReBAR repository onto your Amazon EC2 FireSim manager instance.
+FireSim currently supports running only on Amazon EC2 F1 FPGA-enabled virtual instances on the public cloud.
+In order to simulate your REBAR design using FireSim, you should follow the following steps:
+
+Follow the initial EC2 setup instructions as detailed in the `FireSim documentation  <http://docs.fires.im/en/latest/Initial-Setup/index.html>`__.
+Then clone your full REBAR repository onto your Amazon EC2 FireSim manager instance.
```
Comment (jerryz123) [inline docs/Simulation/FPGA-Based-Simulators.rst]: Is there really a set of steps to run arbitrary REBAR stuff in FireSim already?

File: docs/Simulation/Commercial-Simulators.rst [hunk 2]
```diff
@@ -1,33 +1,39 @@
...
-The ReBAR framework currently supports only the VCS commerical simulator
 
 VCS
 -----------------------
-VCS is a commercial RTL simulator developed by Synopsys. It requires commerical licenses.
-The ReBAR framework can compile and execute simulations using VCS. VCS simulation will generally compile
-faster than Verilator simulations.
+
+`VCS <https://www.synopsys.com/verification/simulation/vcs.html>`__ is a commercial RTL simulator developed by Synopsys.
+It requires commerical licenses.
```
Comment (jerryz123) [inline docs/Simulation/Commercial-Simulators.rst]: Spelling

File: docs/Getting-Started/REBAR-Basics.rst [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+**Rocket**
+  An in-order RISC-V core.
+  See :ref:`Rocket` for more information.
+
+**BOOM (Berkeley Out-of-Order Machine)**
+  An out-of-order RISC-V core.
+  See :ref:`Berkeley Out-of-Order Machine (BOOM)` for more information.
+
+Data-Parallel Accelerators
```
Comment (jerryz123) [inline docs/Getting-Started/REBAR-Basics.rst]: Let's just say "Accelerators" here.

File: docs/Simulation/FPGA-Based-Simulators.rst (hunk 1 above)
Comment (abejgonzalez) [reply docs/Simulation/FPGA-Based-Simulators.rst]: No this is just a temp. placeholder until FireSim comes online within the repository.

**Pull request merged**